pages/Magic-Workbench.md

```diff
@@ -1,12 +1,15 @@
-The Magic Workbench is a [Basic Machine](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machine) in Slimefun.<br>
-It is used to craft many [magical items](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Items), [gadgets](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Gadgets) and even [armor](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Armor).
+The Magic Workbench is a [Basic Machine](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machine) in Slimefun. It is a multiblock structure.<br>
 
 ## Obtaining
-The Magic Workbench is a Multiblock structure which consists of a Bookshelf, Crafting Table and Dispenser placed side by side.<br>
-A complete Magic Workbench looks like this :
+The Magic Workbench is a multiblock structure which consists of a bookshelf, crafting table and dispenser placed side by side.<br>
 
-![Magic Workbench setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/images/multiblock-magic-workbench.png)
+A complete setup should look like this:
+
+![Magic Workbench Setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/images/multiblock-magic-workbench.png)
```

Comment:
There is a 'master' missing in the path (right between the repository name and 'images').
That is probably also why the image isn't showing.

Als, please revert to the previous magic workbench image, I think it's more cleaner than your blue background one.